Src/Tracer.InterfacedObject.pas

```diff
@@ -28,16 +34,22 @@ implementation
 
 uses
 {$IFDEF DEBUG}
-  Winapi.Windows,
+  Windows,
   Tracer.Logger,
 {$ENDIF}
-  System.SysUtils;
+  SysUtils;
 
 { TTracerInterfacedObject }
 
 function TTracerInterfacedObject._AddRef: Integer;
 begin
-  Result := AtomicIncrement(FRefCount);
+  FCriticalSection.Enter;
```

Comment:
Yep, your are right. But anyway this is only for debugging purposes ^_^ See last commit